File: frontend/src/api/department_position.js
import http from './http'

export const getDepartmentPositions = (params) => http.get('/department-positions', { params })
export const createDepartmentPosition = (data) => http.post('/department-positions', data)
export const deleteDepartmentPosition = (id) => http.delete(`/department-positions/${id}`)

File: backend/routes/department_position.go
package routes

import (
	"oa-system/handlers"

	"github.com/gin-gonic/gin"
)

// departmentPositionRoutes 执行相关业务逻辑
func departmentPositionRoutes(rg *gin.RouterGroup) {
	rg.GET("/department-positions", handlers.GetDepartmentPositions)
	rg.POST("/department-positions", handlers.CreateDepartmentPosition)
	rg.DELETE("/department-positions/:id", handlers.DeleteDepartmentPosition)
}
